### ISSUE TITLE
Python3 formatting fail when passing None

### ISSUE THREAD

**Issue by rgayon** [google/turbinia#438]
There might be quite a bunch of instances of this
```
turbiniactl -j help
[INFO] Turbinia version: 20190819
Traceback (most recent call last):
  File "/home/romaing/venvs/turbinia-dev3/bin/turbiniactl", line 11, in <module>
    load_entry_point('turbinia==20190819', 'console_scripts', 'turbiniactl')()
  File "/home/romaing/venvs/turbinia-dev3/lib/python3.5/site-packages/turbinia-20190819-py3.5.egg/turbinia/turbiniactl.py", line 597, in main
    log.warning('Command {0:s} not implemented.'.format(args.command))
TypeError: non-empty format string passed to object.__format__
```

**Comment by rgayon** [comment]
Another place:
       
turbinia/workers/__init__.py:
   self.log(
              'Evidence {0:s} has empty or missing file at source_path {1:s} so '
              'not saving.'.format(evidence.name, evidence.source_path))

**Comment by alimez** [comment]
>     'Saved copyable evidence data to {0:s}'.format(evidence_.saved_path))                                     │[INFO] 0 Tasks still outstanding.
> TypeError: unsupported format string passed to NoneType.__format__                                            │[INFO] 0 Tasks still outstanding.
>

**Comment by aarontp** [comment]
These should be fixed by #458 , but if you notice any other occurrences, please reopen (or file new issues).
